NewSpring/ApollosProject
# Adds amplitude integration via react-native-amplitude-analytics

## DESCRIPTION

### What does this PR do, or why is it needed?

Adds event tracking via Amplitude to the React Native project. 

### How do I test this PR?

1. Clone this PR.
2. Run the app, rebuild for iOS or Android
 - You will need to add a new `.env` variable, Vincent or Isaac should have it.
3. Click around the app. 
4. Check for new events in the Amplitude Dashboard. Vincent or Isaac can provide access.

---

> I am affirming this is my _best_ work ([Ecclesiastes 9:10](https://www.bible.com/bible/97/ECC.9.10.MSG))

## TODO


## REVIEW

**Manual QA**


**Code Review: Questions to consider**


---

> The purpose of PR Review is to _improve the quality of the software._
